lib/errors/error_handling.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module ErrorHandling
```

Comment:
This is specifically about handling API errors, so I would call it that: `ApiErrorHandling`.

Reply:
Done!